WordPress/src/main/java/org/wordpress/android/models/JetpackSettingsModel.java

```diff
@@ -1,17 +1,27 @@
 package org.wordpress.android.models;
 
+import java.util.ArrayList;
+
 public class JetpackSettingsModel {
     public long localTableId;
     public boolean monitorActive;
     public boolean emailNotifications;
     public boolean wpNotifications;
+    public boolean jetpackProtectEnabled;
+    public final ArrayList<String> jetpackProtectWhitelist = new ArrayList<>();
 
     @Override
     public boolean equals(Object other) {
         if (!(other instanceof JetpackSettingsModel)) return false;
         JetpackSettingsModel otherModel = (JetpackSettingsModel) other;
         return monitorActive == otherModel.monitorActive &&
                 emailNotifications == otherModel.emailNotifications &&
-                wpNotifications == otherModel.wpNotifications;
+                wpNotifications == otherModel.wpNotifications &&
+                jetpackProtectEnabled == otherModel.jetpackProtectEnabled &&
+                whitelistMatches(otherModel.jetpackProtectWhitelist);
+    }
+
+    public boolean whitelistMatches(ArrayList<String> otherWhitelist) {
+        return otherWhitelist != null && jetpackProtectWhitelist.equals(otherWhitelist);
```

Comment:
I agree that Set comparisons make more sense and I've updated this method to do that. However, the way list editing is handled currently (blacklist, whitelist) require a List (see `mEditingList`) so I'm going to keep the field as a List for now.